calico/felix/config.py

```diff
@@ -304,9 +310,11 @@ def _finish_update(self, final=False):
         :param final: Have we completed (rather than just read env and config
                       file)
         """
-        self.ETCD_ADDR = self.parameters["EtcdAddr"].value
+
         self.HOSTNAME = self.parameters["FelixHostname"].value
         self.ETCD_SCHEME = self.parameters["EtcdScheme"].value
+        self.ETCD_ADDR = self.parameters["EtcdAddr"].value
```

Comment:
This is the line I was talking about.  Why do we still assign to `self.ETCD_ADDR`?